.editorconfig

```diff
@@ -129,7 +129,7 @@ csharp_style_prefer_readonly_struct_member = true
 
 # Code-block preferences
 csharp_prefer_braces = true
-csharp_prefer_simple_using_statement = true
+csharp_prefer_simple_using_statement = false
```

Comment:
This doesn't prevent simple using statements but allows using old style ones with braces without getting squiggles.  Clearly demonstrating when the using goes out of scope (and thus the dispose operation runs) is sometimes important and I think it should be left to developer discretion to choose which syntax creates the most readable code.  There may also be performance considerations (see examples below). Thoughts?

Examples:
Simple:
```
private Task<Something> MethodAsync()
{
    using var transaction = await dbContext.Database.BeginTransactionAsync(IsolationLevel.Serializable, ct);
    // fetch data from the DB
    // save data in the DB

    // do some things in memory

    // make a web request

    return something;

    // transaction goes out of scope here due to implicit using scope and all above operations took place in that transaction include extraneous web request which needlessly lock rows in the DB for longer than needed
}
```
Old style:
```
private Task<Something> MethodAsync()
{
    using (var transaction = await dbContext.Database.BeginTransactionAsync(IsolationLevel.Serializable, ct))
    {
        // fetch data from the DB
        // save data in the DB

        // transaction goes out of scope here due to explicit closing brace
    }

    // do some things in memory

    // make a web request

    return something;
}
```

Reply:
Yeah, this makes sense.